### ISSUE TITLE
Proton updated to 3.16-7, However DXVK_HUD displays version 0.94

### ISSUE THREAD

**Issue by 0x221E** [ValveSoftware/Proton#2337]
The steam client shows Proton version 3.16-7 Beta. However DXVK_HUD version shows version to be 0.94.

`/dist/lib/wine/dxvk/version` file of **`Proton 3.16 Beta`** directory shows -

`3b3ccc8194733e20ce7e2a54edcf54fe4a2b9785 dxvk (v0.94-18-g3b3ccc8)`

Is there any manual intervention required to update the version ?

**Comment by chui2ch** [comment]
manually download it through steal tools. I did and it worked.

**Comment by benjamimgois** [comment]
Same problem here. Still stucked in dxvk 0.94

**Comment by Bleuzen** [comment]
Ok, let's see everything together:
 - DXVK is still on 0.94 (as with Proton 3.16-6), but it should be newer as you can see in changelog for 3.16-7
 - Into the breach fullscreen is not fixed (https://github.com/ValveSoftware/Proton/issues/2338), but it should be fixed as in changelog for 3.16-7
 - Uplay is still not working (as with 3.16-6), but it should be fixed with 3.16-7

So to mee it seems like someone did a mistake and built Proton 3.16-6 again, but named it 3.16-7.

**Comment by Brisse89** [comment]
Happened to me as well.

Steam said it was using 3.16-7 but in reality it was running on an older version which I noticed when using 'DXVK_HUD=version'.

Installing Proton from the tools menu worked around the issue.

Deleting the Proton folders also probably works around the issue but Steam then re-downloaded my entire games which is a lot of data and takes a lot of time.

Note that whitelisted games are usually running whatever version Valve testers used when whitelisting it, but even when I used the game specific settings to override that with 3.16-7, it was in reality still running on the old version until I applied one of the workarounds.

**Comment by Brisse89** [comment]
> Ok, let's see everything together:
> 
>     * DXVK is still on 0.94 (as with Proton 3.16-6), but it should be newer as you can see in changelog for 3.16-7
> 
>     * Into the breach fullscreen is not fixed (#2338), but it should be fixed as in changelog for 3.16-7
> 
>     * Uplay is still not working (as with 3.16-6), but it should be fixed with 3.16-7
> 
> 
> So to mee it seems like someone did a mistake and built Proton 3.16-6 again, but named it 3.16-7.

No, I think the issue is that the Steam-client doesn't update your Proton even though it says it's running the latest. Apply one of the workarounds mentioned earlier to get the latest version.

**Comment by Bleuzen** [comment]
@Brisse89 Thanks, you are right. (Re)downloading Proton 3.16 from the tools menu fixed it.

**Comment by Brisse89** [comment]
My guess is there has been some regression in how the Steam client handles Proton updates after the zero-byte updates fix. This is probably a bug in [ValveSoftware/steam-for-linux](https://github.com/ValveSoftware/steam-for-linux) rather than a Proton bug.

**Comment by Brisse89** [comment]
Possible duplicate, or possibly related to this issue: https://github.com/ValveSoftware/steam-for-linux/issues/6045
